jsontests/src/in_memory.rs

```diff
@@ -142,6 +153,17 @@ impl RuntimeBaseBackend for InMemoryBackend {
 			.unwrap_or(H256::default())
 	}
 
+	fn transient_storage(&self, address: H160, index: H256) -> H256 {
+		self.state
```

Comment:
With one global hashmap you can replace this instruction by: 

```
		self.transient_storage
            .get(&(address, key))
            .copied()
            .unwrap_or_default()
```

Reply:
Replied in a thread above